uint32 motor
uint32 angle
---
bool result
